registry/handlers/app.go

```diff
@@ -640,6 +640,7 @@ func (app *App) dispatcher(dispatch dispatchFunc) http.Handler {
 		}
 
 		context := app.context(w, r)
+		defer defaultContextManager.release(context)
```

Comment:
It should be safe to do multiple releases, my main concern is that if there was a memory leak discovered are we sure this fixes it.